.circleci/config.yml

```diff
@@ -210,7 +210,8 @@ jobs:
           export TM_ENVIRONMENT=<< parameters.stack_name >>
           yarn
           CI=true GENERATE_SOURCEMAP=false yarn build
-          aws s3 sync build/ s3://tasking-manager-<< parameters.stack_name >>-react-app --delete
+          aws s3 sync build/ s3://tasking-manager-<< parameters.stack_name >>-react-app --delete --cache-control max-age=31536000
+          aws s3 cp s3://tasking-manager-<< parameters.stack_name >>-react-app s3://tasking-manager-<< parameters.stack_name >>-react-app --recursive --exclude "*" --include "*.html" --metadata-directive REPLACE --cache-control max-age:no-cache --content-type text/html
```

Comment:
Cache control parameter must be set to `no-cache` like so:

`--cache-control no-cache`

Reply:
@dakotabenjamin Need this change to happen before merge.